From d06c3e59c01b9c15d35464f02d163d1ef07abe8f Mon Sep 17 00:00:00 2001
From: "Kirill A. Korinsky" <kirill@korins.ky>
Date: Thu, 10 Feb 2022 00:53:41 +0100
Subject: [PATCH] `libssh2-sys`: use `src` instead `.git` as vendored indicator

When someone vendored `libssh-sys` he may exclude `.git` folder.

When such things happened an attempt to build it may lead to error like:
`fatal: not a git repository (or any of the parent directories): .git`.
---
 vendor/libssh2-sys/build.rs | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/vendor/libssh2-sys/build.rs b/vendor/libssh2-sys/build.rs
index 215014864b..235bc16cd7 100644
--- a/vendor/libssh2-sys/build.rs
+++ b/vendor/libssh2-sys/build.rs
@@ -32,7 +32,7 @@ fn main() {
         }
     }
 
-    if !Path::new("libssh2/.git").exists() {
+    if !Path::new("libssh2/src").exists() {
         let _ = Command::new("git")
             .args(&["submodule", "update", "--init"])
             .status();
-- 
2.37.2

